src/ORM/SS_List.php

```diff
@@ -83,6 +83,14 @@ public function find($key, $value);
      */
     public function column($colName = "ID");
 
+    /**
+     * Returns a unique array of a single field value for all items in the list.
+     *
+     * @param  string $colName
+     * @return array
+     */
+    public function columnUnique($colName = "ID");
```

Comment:
I'm afraid we can't change the interface in a minor release because this will break any other classes that implement it. We'd have to add this in 5.x

Reply:
It's a new method not a change isn't it?

Personally I like `distinctColumn` instead of unique to be consistent, but I'm easy really.

Reply:
Adding it to the interface would cause a fatal error for any classes that implement the interface until they also implement the new method